src/components/Home/WhatsNewModal/styles.module.css

```diff
@@ -0,0 +1,165 @@
+.container {
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  z-index: 11;
+
+  &:hover {
+    cursor: pointer;
+  }
+
+  &Open {
+    height: 100%;
+    width: 100%;
+    background: rgba(132, 142, 148, 0.5);
+  }
+}
+
+.button {
+  @mixin button-big;
+
+  position: absolute;
+  box-sizing: border-box;
+  overflow: hidden;
+  white-space: nowrap;
+  left: 30px;
+  bottom: 30px;
+  z-index: 3;
+  width: 154px;
+  height: 62px;
+  padding: 16px 24px;
+  background: linear-gradient(
+    102.58deg,
+    #13adc7 22.89%,
+    #17d5f5 24.16%,
+    #13adc7 25.62%
+  );
+  background-size: 600% auto;
+  color: #fff;
+  border: none;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 99px;
+  font-family: var(--font-brandon);
+  transition: all 0.25s ease-in-out;
+  animation: gradient 3s infinite ease-in-out;
+
+  @media (--xs-scr) {
+    left: 15px;
+    bottom: 15px;
+  }
+
+  &::before {
+    position: absolute;
+    content: '';
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    z-index: -1;
+    opacity: 0;
+    border-radius: 99px;
+    transition: opacity 0.2s ease-out;
+    background: var(--color-azure-hover);
+  }
+
+  &:hover {
+    cursor: pointer;
+
+    &::before {
+      opacity: 1;
+    }
+  }
+
+  &Exit {
+    border-radius: 50%;
+    padding: 14px;
+    width: 48px;
+    height: 48px;
+    background: var(--color-gray-dark);
+    box-shadow: 0 1px 2px rgba(46, 49, 55, 0.1), 0 0 1px rgba(46, 49, 55, 0.5);
+
+    &:hover {
+      background: var(--color-gray);
+
+      &::before {
+        opacity: 0;
+      }
+    }
+
+    svg {
+      fill: currentColor;
+
+      path {
+        fill: currentColor;
+      }
+    }
+  }
+}
+
+@keyframes gradient {
+  0% {
+    background-position: center;
+  }
+
+  66% {
+    background-position: right center;
+  }
+}
+
+.modal {
+  box-sizing: border-box;
+  position: absolute;
+  left: 62px;
+  bottom: 62px;
+  background: #fff;
+  box-shadow: 0 8px 16px rgba(46, 49, 55, 0.1), 0 0 2px rgba(46, 49, 55, 0.2);
+  width: 460px;
+  border-radius: 16px;
+  padding: 20px;
+  opacity: 0;
+  visibility: hidden;
+  transition: opacity 0.2s ease-in-out, visibility 0.2s ease-in-out;
+
+  &:hover {
+    cursor: initial;
+  }
+
+  @media (--xs-scr) {
```

Comment:
I went ahead and adjusted the model and button positions slightly so the modal could atleast fit inside phone screens.

Reply:
Today at the meeting we discussed the contents of the modal:
1. This will be the link: https://dvc.org/blog/ml-experiment-versioning
2. Change Header to "ML Experiment Versioning"
3. Words to: "Versioning machine learning experiments combines the benefits of version control and experiment tracking. Read more."
4. Add pic from blog post into modal: Title, then pic, then text 

(cc: @dberenbaum for further confirmation)